pyproject.toml

```
@@ -0,0 +1,41 @@
+[tool.poetry]
+name = "planemo"
+version = "0.75.0"
+description = "Command-line utilities to assist in building tools for the Galaxy project (http://galaxyproject.org/)."
+authors = ["Galaxy Project and Community"]
+license = "MIT"
+
+[tool.poetry.dependencies]
+python = "^3.6"
+allure-python-commons = "*"
+bioblend = ">=0.14.0"
```

```suggestion
bioblend = ">=0.16.0"
```

xref. https://github.com/galaxyproject/planemo/pull/1140#pullrequestreview-797695008